tldw_Server_API/app/core/Chatbooks/services/jobs_worker.py

```
@@ -194,12 +194,12 @@ async def _handle_import(service: ChatbookService, payload: Dict[str, Any], job_
 
     selections = _map_content_selections(payload.get("content_selections") or {})
     conflict_resolution = _parse_conflict_resolution(payload.get("conflict_resolution", "skip"))
-    file_path = payload.get("file_path")
-    if not file_path or not str(file_path).strip():
-        raise ChatbooksJobError("Missing file_path for import job", retryable=False)
+    file_ref = payload.get("file_token") or payload.get("file_path")
+    if not file_ref or not str(file_ref).strip():
+        raise ChatbooksJobError("Missing file reference for import job", retryable=False)
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)